backend/app/repositories/helpOffers/update.ts

```diff
@@ -0,0 +1,26 @@
+import db from "../../db/connection";
+
+import { HelpOffer } from "../../db/seeds/data/types/data.types";
+
+export const update = async (user_id: string, helpOfferBody:any): Promise<HelpOffer> => {
+    const {help_request_id, status} = helpOfferBody
+
+    const values = [status, user_id, help_request_id]
+
+    if(values.length ===0){
+        const {rows} = await db.query(
+           `SELECT helper_id, help_request_id, status, users.first_name, users.last_name, users.additional_contacts, users.post_code FROM help_offers LEFT JOIN users on users.id = help_offers.helper_id WHERE helper_id =$1, help_request_id=$2`, [user_id, help_request_id]
+        )
+        return rows[0]
```

Comment:
I don't get the point of returning the user? it's part of validation logic which we can implement later in the service

Reply:
Don't we want to see all that re the UX/Ui design?

Reply:
Ahh yeah,  we'll have a look at it later